packages/create-contentful-app/lib/create-app-definition.js

```diff
@@ -0,0 +1,94 @@
+/* eslint-disable no-console, no-process-exit */
+
+const inquirer = require('inquirer');
+const path = require('path');
+const { createClient } = require('contentful-management');
+const login = require('./login');
+
+async function fetchOrganizations(client) {
+  try {
+    const orgs = await client.getOrganizations();
+
+    return orgs.items.map(org => ({
+      name: `${org.name} (${org.sys.id})`,
+      value: org.sys.id
+    }));
+  } catch (err) {
+    console.log();
+    console.log(
+      'We couldnt find your organizations. Please make sure you provided an access token'
```

Comment:
`Could not fetch your organizations. Make sure you've provided a valid access token.`